src/cookbooks/looker/intro.md

```diff
@@ -0,0 +1,62 @@
+# Introduction to Looker
+
+In 2020, Mozilla chose [Looker](https://looker.com/) as its primary tool for analyzing data.
+It allows data exploration and visualization by experts and non-experts alike.
+
+Access to Looker is currently limited to Mozilla employees and designated contributors. For more information, see [gaining access](../../concepts/gaining_access.md).
+
+## Table of Contents
+
+<!-- toc -->
+
+## Accessing Looker
+
+You can access Mozilla's instance of Looker at [`mozilla.cloud.looker.com`](https://mozilla.cloud.looker.com).
+
+## Getting Started
+
+### Front page
+
+By default, on the front page you will see a list of default folders, which contain links to dashboards.
+These are organized by project.
+Of particular note is the "Mozilla Metrics" folder, which includes several Data-produced and vetted dashboards like the Firefox Corporate KPI Dashboard.
+
+![](../../assets/Looker_screenshots/front-page.png)
+
```

Comment:
I wonder if we should mention namespaces (=functional group represented within Looker) here? Since namespaces essentially result in new folders getting added? And maybe link to https://mozilla-hub.atlassian.net/jira/software/c/projects/LOOK/issues for requesting new namespaces?

Reply:
I think the namespaces concept is probably better introduced in a reference document. Let's do that as a followup